CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [*Ongoing*](https://github.com/pbrisbin/downgrade/compare/v6.3.0...master)
 
+- [FIX] Remove --arch option and corresponding logic due to redundancy
 - [NEW] Handling hard dependencies during downgrading
 
 ## [*v6.3.0*](https://github.com/pbrisbin/downgrade/compare/v6.1.0...v6.3.0)
```

Comment:
There might be a merge conflict here given that version headers are bolded, but in the previous PR they are not.